packages/components/src/components/form/SelectBar/SelectBar.tsx

```diff
@@ -169,38 +173,46 @@ export const SelectBar: <V extends ValueTypes>(props: SelectBarProps<V>) => JSX.
     const handleKeyboardNav = (e: KeyboardEvent) => {
         const selectedOptionIndex = options.findIndex(option => option.value === selectedOptionIn);
 
+        let option;
         switch (e.key) {
```

Comment:
nitpick: I would rather use normal `if` instead of `switch` and maybe ternary in inner condition (but `[option]` could be a bit tricky). 
Anyway we don't use any of the feature that switch offers: multiple options, defaut value, returning values etc.

Reply:
good one! 4aeae0a3b2